### ISSUE TITLE
Opening a notebook from markers view is broken

### ISSUE THREAD

**Issue by jrieken** [microsoft/vscode#106657]
* open GH issues notebook
* make sure to have some errors
* select an error from the markers view
* 🐛 the cell opens as readonly editor, not as notebook

This happens because an early return here ([we have no editor id and the input reports as being untitled]( https://github.com/microsoft/vscode/blob/f3532c9061bb22e3423f21fb0b89ee3c1e76c001/src/vs/workbench/contrib/notebook/browser/notebook.contribution.ts#L291-L295)). The original input being untitled seems to have changed recently via 183d11a642b121c3023d3f1c9f2470263c1452bc. fyi @bpasero this breaks our assumption and might have more impact in places which aren't known yet.

**Comment by jrieken** [comment]
fyi @rebornix - there might more usages that only you know of

**Comment by bpasero** [comment]
@jrieken yeah this was a recent change to allow saving readonly editors. Conceptually readonly editors (coming from virtual document providers) are untitled until you save them. This was a necessary change to handle the transition from saving the editor to opening it again after saving possible. 

If you must check for untitled editors that are writeable, looking for `untitled:` scheme seems legit to me. 

When I did this change I checked for all users of `isUntitled` and did not find issues, though I probably missed this one.
